backend/views/views.py

```diff
@@ -149,16 +145,16 @@ def login():
     if form.validate_on_submit():
         user = db.session.query(User).filter(
             User.email == form.email.data).first()
-        if user.check_password(form.password.data):
+        if user and not user.delete_date and user.check_password(form.password.data):
             session['user_id'] = user.id
+            session['auth'] = True
```

Comment:
You don't need separate field to indicate whether user is authenticated, you may just check presence of `user_id` in the session, as it already implemented in `admin_permission` decorator.

Reply:
agree, additional useless code